compiler/rustc_mir/src/transform/multiple_return_terminators.rs

```diff
@@ -14,6 +14,12 @@ impl<'tcx> MirPass<'tcx> for MultipleReturnTerminators {
             return;
         }
 
+        if !tcx.consider_optimizing(|| {
```

Comment:
This could also be moved into a loop below to allow individual return uplifts to be bisected